src/PyScriptEC/PyScriptEncConverter.cs

```diff
@@ -129,13 +129,16 @@ bool                bForward
                 // We could use UTF-8 here, but wide data works just fine.
                 // the windows version definitely needs UTF16, but for some reason _MSC_VER is not defined on windows (as I think Jim is expecting)
                 //  so I'll use the opposite logic of 'if not mono'...

```

Comment:
The comment is slightly outdated now. Could be something like

```
// We could use UTF-8 here, but wide data works just fine. The windows version definitely needs UTF16.
```

Reply:
Right, thank you.

Reply:
Done